tools/cache_zenodo_svg.py

```diff
@@ -144,4 +144,3 @@ def _get_xdg_cache_dir():
             )
         fout.write("\n\n")
         fout.write("\n".join(footer))
-        fout.write('\n')
```

Comment:
This newline is stripped by pre-commit hook anyway, and as far as I can tell is the only reason why it was said to manually edit `citing.rst`

If I'm wrong about that, then I didn't know what I _should_ edit there, so that should be clarified.

If I'm right, then the removal I did seems sufficient (still indicating the file in the text above, but removing the manual edit from the script)

Reply:
You adjusted some language around that, but specifically, the path was not updated from `doc/citing.rst` to `doc/users/project/citing.rst` and the `$EDITOR doc/citing.rst` remained in the code block, suggesting that _something_ needed manual editing.

Those edits have already been made here in this PR, but the extra newline at the end of the file was removed by pre-commit hook anyway, so I just made it not generate it in the first place.

Reply:
No, I was referring to the extra newline; it should have been removed by https://github.com/matplotlib/matplotlib/pull/25023/files#diff-853c166124204e03d3443736d47b67ad06e4e0e44c6f49a5b8bfaaf3fdba4c9bR128

Reply:
Ahh... yes, #25023 was not actually backported to the `v3.7.x` branch, so pre-commit caught it there... 

(Which begs the question of whether we used the older github_stats.py for the 3.7.0 release, as that was also in that PR)

So this is rivaling fixes for the same thing, which is an overcorrection... I'll undo this one

Reply:
> (Which begs the question of whether we used the older github_stats.py for the 3.7.0 release, as that was also in that PR)

You did; it still has the generic "GitHub statistics" title instead of the one with the version. But there's not really much other functional difference otherwise.